vm/src/api/ser.rs

```diff
@@ -110,13 +110,13 @@ let f e =
 Compiler::new()
 #   .implicit_prelude(false)
     .load_script(&thread, "test", expr)
-    .unwrap_or_else(|err| panic!("{}", err));
+    .unwrap_or_else(|err| ice!("{}", err));
 
 let mut f: FunctionRef<fn (Ser<Enum>) -> i32> = thread
     .get_global("test.f")
-    .unwrap_or_else(|err| panic!("{}", err));
+    .unwrap_or_else(|err| ice!("{}", err));
 
-let result = f.call(Ser(Enum::B("".to_string(), 4))).unwrap_or_else(|err| panic!("{}", err));
+let result = f.call(Ser(Enum::B("".to_string(), 4))).unwrap_or_else(|err| ice!("{}", err));
```

Comment:
These are doc tests so they should still be panic! (since they display an example on what to write).